src/main/java/com/pgs/booking/model/dto/CreateUpdatePassengerDto.java

```diff
@@ -1,27 +1,32 @@
 package com.pgs.booking.model.dto;
 
-import lombok.Value;
+import lombok.AllArgsConstructor;
 import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
 import javax.validation.constraints.Email;
 import javax.validation.constraints.NotBlank;
 import javax.validation.constraints.Size;
 
-@Value
+@Data
 @Builder
+@NoArgsConstructor
+@AllArgsConstructor
 public class CreateUpdatePassengerDto {
     @NotBlank(message = "Please enter your firstName.")
     @Size(min = 3, max = 45, message = "Passenger firstName should have at least 3 characters.")
```

Comment:
### HINT
Let's not worry about that anymore, but a better approach is to specify validations in more atomic way, because this way you would have confusing message for too long input. 
For example: instead of 
```
@Size(min = 3, max = 45, message = "Passenger firstName should have at least 3 characters.")
```
better would be:
```
@Size(min = 3, message = "Passenger firstName should have at least 3 characters.")
@Size(max = 45, message = "Passenger firstName should have at most 3 characters.")
```

Those annotations are annotated with `@Repeatable` so can be used multiple times.